CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` when updating to this version!
 
+- Bugfix: Messages are now trimmed to the Twitch character limit. (#1448)
 - Minor: Regex banphrases now support more complex regex features. (#1469)
```

Comment:
```suggestion
- Minor: Regex banphrases now support more complex regex features. (#1469)
- Bugfix: Messages are now trimmed to the Twitch character limit. (#1494)
```

Links to PR instead of issue now, and sorts the changelog entries by category